app/services/jwt_token.rb

```diff
@@ -20,6 +20,7 @@ def prepare_payload(user, secret, expiration)
       }
 
       payload[:exp] = (Time.now.to_i + expiration) if expiration
+      payload[:scope] = scope if scope
```

Comment:
This should probably be 
```suggestion
      scope = scope.join(' ') if scope.is_a? Array
      payload[:scope] = scope if scope
```
to comply with the spec.
Also, should we verify the user generating the token actually has the permission to do the action defined in the scope?

Reply:
> Also, should we verify the user generating the token actually has the permission to do the action defined in the scope?

Right now users cannot generate tokens by them self, so they can't choose wich scope will be in the token.

It's up to developers to choose correct scope for the token, and if they'll choose an incorrect scope, we will catch it later anyway - during  authorization with user's roles & filters.

So the current behavior is: They (DEVs) can choose whatever scope they want, but the final word has user's permissions.